src/main/webapp/components/workspaces/workspaces.js

```diff
@@ -258,6 +260,39 @@ const workspaceAttributes = gql`
     metacard_type
   }
 `
+const useClone = () => {
+  const cloneWorkspace = gql`
+    mutation CloneWorkspace($id: ID!) {
+      cloneMetacard(id: $id) {
+        ...WorkspaceAttributes
+        id: id
+        owner: metacard_owner
+        modified: metacard_modified
```

Comment:
I think we should add these attributes to the Workspace gql fragment

Reply:
we might want to create a separate issue to clean up all the graphql in this class; they are scattered around and we make it so we dont have to set `metacard_type` differently than other attributes